api/location/models.py

```diff
@@ -22,16 +22,13 @@ class TimeZoneType(enum.Enum):
 
 class Location(Base, Utility):
     __tablename__ = 'locations'
-    id = Column(Integer, Sequence('floors_id_seq', start=1, increment=1), primary_key=True) # noqa
+    id = Column(Integer, Sequence('locations_id_seq', start=1, increment=1), primary_key=True)  # noqa
```

Comment:
Hello @bevkololi , Kindly remove this #noqa. You can do this by
```
id = Column(Integer, 
Sequence('locations_id_seq', start=1, increment=1),
primary_key=True)
```

Reply:
Thanks @nakatuddesuzan, on it.